ci-operator/config/openshift/csi-driver-smb/openshift-csi-driver-smb-master.yaml

```diff
@@ -0,0 +1,72 @@
+base_images:
+  commitchecker:
+    name: commitchecker
+    namespace: ci
+    tag: "4.15"
+  ocp_4.16_base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.21-openshift-4.16
+binary_build_commands: make smb
+build_root:
+  from_repository: true
+canonical_go_repository: github.com/kubernetes-csi/csi-driver-smb
+images:
+- dockerfile_path: Dockerfile.openshift
+  inputs:
+    ocp_4.16_base:
+      as:
+      - registry.ci.openshift.org/ocp/4.16:base
+    ocp_builder_rhel-8-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.16
+  to: csi-driver-smb
+promotion:
+  to:
+  - name: "4.16"
+    namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.16"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.16"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: GOFLAGS="-mod=readonly" make test
+  container:
+    from: src
+- as: verify-deps
+  steps:
+    test:
+    - ref: go-verify-deps
+- as: security
```

Comment:
Consider setting `always_run: false` on this job, like in [gcp-filestore](https://github.com/openshift/release/blob/8127b65ef7749d08e7607223fd7aab0fc47c23d6/ci-operator/config/openshift/gcp-filestore-csi-driver/openshift-gcp-filestore-csi-driver-master.yaml#L58). I see one issue reported in `pkg/smb/nodeserver.go` that we should probably find some way to address, and IMO it's better not to run the job by default until we can get it to pass.